utils/storage/cookie.js

```diff
@@ -20,9 +20,11 @@ class CookieLocal {
    */
   options(options = {}) {
     if (arguments.length === 0) return this._options;
-
     let domain = `.${topDomain(window.location.href)}`;
     if (domain === ".") domain = null;
+    if (this.get("custom_cookie_domain")) {
+      domain = this.get("custom_cookie_domain");
+    }
```

Comment:
domain = this.get("custom_cookie_domain") || null;

Reply:
but if this.get("..") is not there it will not enter the if